packages/server/src/api/routes/tests/couchTestUtils.js

```diff
@@ -37,6 +41,31 @@ exports.createModel = async (request, instanceId, model) => {
   return res.body
 }
 
+exports.createRecord = async (request, instanceId, modelId, record) => {
```

Comment:
NAB: If you have a lot of arguments, it's almost always better to use an object.

```js
exports.createRecord = async ({ request, instanceId, modelId, record })
```
A few benefits:
- You don't have to pass them in order
- You can have some undefined